### PR TITLE
feat(utils): add configurable color steps

### PR DESCRIPTION
**Pull request by beatrizip**
Taking out the color steps of the method we can configure them in each vertical.

### CODE REVIEW

File: src/utils/_colors.scss [hunk 1]
```diff
@@ -12,25 +12,25 @@
 // $c-primary-dark: color-variation($c-primary, -2) -> 2 steps darker
 // $c-primary-darkest: color-variation($c-primary, -4) -> 4 steps darker
 
+$step-variations-positive: 25, 50, 75, 90;
```
Comment (jcger) [inline src/utils/_colors.scss]: this should have !default at the end, and maybe have a more descriptive name like color-darken-steps and color-lighten-steps

File: src/utils/_colors.scss [hunk 2]
```diff
@@ -12,25 +12,25 @@
 // $c-primary-dark: color-variation($c-primary, -2) -> 2 steps darker
 // $c-primary-darkest: color-variation($c-primary, -4) -> 4 steps darker
```
Comment (davidbarna) [inline src/utils/_colors.scss]: What is the problem you were facing with current implementation ??
AFAIK step variations are not supposed to be overwitten:
https://paper.dropbox.com/doc/Basic-Styles--AJrkoC13T7nSRdECBbSq37VvAg-0ko2wfsTt1JOQHV6G8cwo#:uid=016947049972010127346188&h2=Color-Palette

File: src/utils/_colors.scss (hunk 1 above)
Comment (beatrizip) [reply src/utils/_colors.scss]: Ok 👍

File: src/utils/_colors.scss (hunk 2 above)
Comment (beatrizip) [reply src/utils/_colors.scss]: Transfer this question to UX. I block the PR until I have an answer.

**Comment by beatrizip** [comment]
@davidbarna The paper and the code has been updated to add a color with 95% luminosity

https://paper.dropbox.com/doc/Basic-Styles--AKV83Ak5n50F5tbzZphptdsQAg-0ko2wfsTt1JOQHV6G8cwo

**Pull request merged**